src/UNL/PersonInfo/SignatureGenerator.php

```diff
@@ -0,0 +1,50 @@
+<?php
```

Comment:
Can you remove all the unnecessary whitespace in this file. The whitespace formatting should match the other files